diff --git a/dist/esm/server/app-render/get-css-inlined-link-tags.js b/dist/esm/server/app-render/get-css-inlined-link-tags.js
index f34aceea90d8f21b6aabab1201be8472b9443590..5060f3561f9c53c234197c3f55ac46ec452215c6 100644
--- a/dist/esm/server/app-render/get-css-inlined-link-tags.js
+++ b/dist/esm/server/app-render/get-css-inlined-link-tags.js
@@ -5,8 +5,8 @@
     const filePathWithoutExt = filePath.replace(/\.[^.]+$/, '');
     const cssChunks = new Set();
     const jsChunks = new Set();
-    const entryCSSFiles = clientReferenceManifest.entryCSSFiles[filePathWithoutExt];
-    const entryJSFiles = ((_clientReferenceManifest_entryJSFiles = clientReferenceManifest.entryJSFiles) == null ? void 0 : _clientReferenceManifest_entryJSFiles[filePathWithoutExt]) ?? [];
+    const entryCSSFiles = clientReferenceManifest?.entryCSSFiles?.[filePathWithoutExt] ?? [];
+    const entryJSFiles = clientReferenceManifest?.entryJSFiles?.[filePathWithoutExt] ?? [];
     if (entryCSSFiles) {
         for (const css of entryCSSFiles){
             if (!injectedCSS.has(css.path)) {
diff --git a/dist/server/app-render/get-css-inlined-link-tags.js b/dist/server/app-render/get-css-inlined-link-tags.js
index fba82f83274b3e7f9c935666727e822be758fe16..fa4d503d293f98288e7f7fe00184224b25822182 100644
--- a/dist/server/app-render/get-css-inlined-link-tags.js
+++ b/dist/server/app-render/get-css-inlined-link-tags.js
@@ -13,8 +13,8 @@ function getLinkAndScriptTags(clientReferenceManifest, filePath, injectedCSS, in
     const filePathWithoutExt = filePath.replace(/\.[^.]+$/, '');
     const cssChunks = new Set();
     const jsChunks = new Set();
-    const entryCSSFiles = clientReferenceManifest.entryCSSFiles[filePathWithoutExt];
-    const entryJSFiles = ((_clientReferenceManifest_entryJSFiles = clientReferenceManifest.entryJSFiles) == null ? void 0 : _clientReferenceManifest_entryJSFiles[filePathWithoutExt]) ?? [];
+    const entryCSSFiles = clientReferenceManifest?.entryCSSFiles?.[filePathWithoutExt] ?? [];
+    const entryJSFiles = clientReferenceManifest?.entryJSFiles?.[filePathWithoutExt] ?? [];
     if (entryCSSFiles) {
         for (const css of entryCSSFiles){
             if (!injectedCSS.has(css.path)) {
